Henadzi_Ulasik/02.Git.Local/02.Git.Local.md

```diff
@@ -0,0 +1,171 @@
+   17  git config user.email "duru_85@mail.ru"
+   18  git config user.email "duru_85@mail.ru"
+   19  git config user.email
+   20  git init
+   21  ls -l .git/
+   22  git status
+   23  vim .git/config
+   24  git config user.name
+   25  git config user.email "duru_85@mail.ru"
+   26  git config user.email
+   27  vim .git/config
+   28  date > first.txt
+   29  cat first.txt
+   30  git status
+   31  vim .gitignore
+   32  git status
+   33  touch .env
+   34  git status
+   35  git add --all
+   36  git status
+   37  git commit -m "Init commit"
+   38  git log --oneline
+   39  date >> first.txt
+   40  date >> first.txt
+   41  date >> first.txt
+   42  git status
+   43  git add -all
+   44  git commit -m "Second commit"
+   45  git add --all
+   46  git commit -m "Second commit"
+   47  git status
+   48  git log --online
+   49  git log --oneline
+   50  vim first.txt
+   51  sudo apt install bash-completion
+   52  git commit --amed -a
+   53  git commit --amend -a
+   54  git log --oneline
+   55  git checkout dev
+   56  git checkout -b dev
+   57  git branch
+   58  date >> second.txt
+   59  date >> second.txt
+   60  date >> second.txt
+   61  date >> second.txt
+   62  git status
+   63  gid add second.txt
+   64  git add second.txt
+   65  git commit -m "Therd commit"
+   66  git status
+   67  git log -oneline
+   68  git log --oneline
+   69  data >> third.txt
+   70  date >> third.txt
+   71  git status
+   72  git add -all
+   73  git add --all
+   74  git commit -m "Fourth commit"
+   75  git log --oneline
+   76  git checkout -b features/do_one
+   77  date >> fifth.txt
+   78  git add --all
+   79  git commit -m "Fifth commit"
+   80  git log --oneline
+   81  git checkount master
+   82  git checkout master
+   83  git log --oneline
+   84  date >> six.txt
+   85  git add -all
+   86  y
+   87  git add --all
+   88  git commit -m "Six commit"
+   89  git log --oneline
+   90  git reset --hard 16a6606
+   91  git log --oneline
+   92  get checkout -b hotfix/we_gonna_die
+   93  git checkout -b hotfix/we_gonna_die
+   94  date >> six.txt
+   95  git add --all
+   96  git commit -m "Six commit"
+   97  git log --oneline
+   98  git checkout master
+   99  git branch
+  100  git rebase features/do_one
+  101  git history
+  102  git --hist
+  103  git --help
+  104  history
+  105  git ctatus
+  106  git status
+  107  git rebase HEAD~1
+  108  git log --oneline
+  109  git rebase HEAD~4
+  110  git log --oneline
+  111  git reset --hard 16a6606
+  112  git log --oneline
+  113  git cheripick 044be96
+  114  git --help
+  115  git cherry-pick 044be96
+  116  git log --oneline
+  117  git branch
+  118  git checkout dev
+  119  git cherry-pick 044be96
+  120  git log --oneline
+  121  git branch
+  122  git checkout features/do_one
+  123  git cherry-pick 044be96
+  124  git log --oneline
+  125  git branch
+  126  history
+```
+
+### The lists of commits for all branches:
+
+* master branch:
+```
+$ git log --oneline
```

Comment:
commits from dev and features/do_one branch are missing in master branch

Reply:
Fixed